packages/houdini/src/lib/config.ts

```diff
@@ -473,13 +473,20 @@ export class Config {
 		return false
 	}
 
+	private pathRmvQueryString(filepath: string) {
+		const parsed = path.parse(filepath)
+		return `${parsed.dir}/${parsed.name}${parsed.ext.split('?')[0]}`
+	}
+
 	includeFile(
 		filepath: string,
 		{
 			root = this.projectRoot,
 			ignore_plugins = false,
 		}: { root?: string; ignore_plugins?: boolean } = {}
 	) {
+		filepath = this.pathRmvQueryString(filepath)
+
```

Comment:
Seems like the extra function doesn't really offer us much. How about we just inline the logic like this?

```suggestion
	includeFile(
		filepath: string,
		{
			root = this.projectRoot,
			ignore_plugins = false,
		}: { root?: string; ignore_plugins?: boolean } = {}
	) {
		const parsed = path.parse(filepath)
		filepath =  `${parsed.dir}/${parsed.name}${parsed.ext.split('?')[0]}`

```

Reply:
It was to convey the why, but it's ok. Let's switch